### ISSUE TITLE
applied the baseline correction？

### ISSUE THREAD

**Issue by fangtao365** [pouya-bashivan/EEGLearn#53]
Hello, dear author, I would like to know whether you have applied the baseline correction method?
In other words, does the average power obtained subtract the average power before the mark?

**Comment by pbashivan** [comment]
yes, to make the power values more comparable across trials it's best to perform the baseline correction on each trial.
